### PR TITLE
lesson11

### PR DESCRIPTION
**Pull request by Maksimilian13**

### CODE REVIEW

File: lesson11/1001fonts-white-love-eula.txt
Comment (RomanIlkiv) [inline lesson11/1001fonts-white-love-eula.txt:1]: why this file is needed?

File: lesson11/2758625-middle.png
Comment (RomanIlkiv) [inline lesson11/2758625-middle.png:1]: move all images to separate folder, for example `img`

File: lesson11/Lobster-Regular.ttf
Comment (RomanIlkiv) [inline lesson11/Lobster-Regular.ttf:1]: if you so many fonts, move them also to separate folder

File: lesson11/OFL.txt
Comment (RomanIlkiv) [inline lesson11/OFL.txt:1]: remove this file

File: lesson11/index11.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                </a>
+            </div>
+            <div>
+                <a class="nav-list-hot-dog" href="">
+                    <img class="nav-list-hot-dog-img" src="hot-dog.jpg" alt="hot-dog">
+                    <span class="nav-list-hot-dog-span">hot-dog</span>
+                </a>
+            </div>
+            <div>
+                <a class="" href="">
```
Comment (RomanIlkiv) [inline lesson11/index11.html:71]: why empty `attribute` class?

File: lesson11/index11.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    <head>
+        <title>Burger_Rectaurant</title>
+        <link rel="stylesheet" href="style11.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <nav>
+           <a href="">Home</a>
```
Comment (RomanIlkiv) [inline lesson11/index11.html:12]: put # everywhere in href for now

without it, browser will not consider this button as a link

File: lesson11/style11.css [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    font-family: whittelove;
+    src: url(White\ Love\ OTF.otf);
+}
+@font-face {
+    font-family: lobster;
+    src: url(Lobster-Regular.ttf);
+}
+@font-face {
+    font-family: merriweather;
+    src: url(Merriweather-BoldItalic.ttf);
```
Comment (RomanIlkiv) [inline lesson11/style11.css:11]: I see you have just 3 fonts in css, why you so many fonts files add to the project?

File: lesson11/style11.css [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+    width: 100%;
+    height: 80px;
+ }
+ header p {
+    color: white;
+ }
+ .number-time {
+    margin-top: 25px;
+    justify-content: center;
+
```
Comment (RomanIlkiv) [inline lesson11/style11.css:46]: delete empty line

**Review by RomanIlkiv — CHANGES_REQUESTED** [review]

**Comment by Maksimilian13** [comment]
https://github.com/Maksimilian13/robot_dreams/pull/6/commits/ee05d6da3f4b1a9827c2736310be226a8b385339

пн, 5 черв. 2023 р. о 08:47 Roman ***@***.***> пише:

**Pull request merged**